README.md

```diff
@@ -20,26 +20,83 @@ The Go library documentation can be found [here][libdocs].
 
 This library supports the following Go implementations:
 
-* Go 1.15
-* Go 1.16
-* Go 1.17
-* Go 1.18
-* Go 1.19
+- Go 1.15
+- Go 1.16
+- Go 1.17
+- Go 1.18
+- Go 1.19
```

Comment:
I know unrelated, but can we add Go 1.20 here? It was released a few months ago.

Reply:
All the things! Done.